near-token-factory/src/lib.rs

```diff
@@ -26,11 +28,21 @@ enum StorageKey {
     TokenMap,
 }
 
+#[derive(AccessControlRole, Deserialize, Serialize, Copy, Clone)]
+#[serde(crate = "near_sdk::serde")]
+pub enum AclRole {
+    UpdateTokenBinary,
+    UpdateTokenAdmin,
+}
+
+#[access_control(role_type(AclRole))]
 #[near_bindgen]
 #[derive(BorshSerialize, BorshDeserialize, PanicOnDefault)]
 pub struct Contract {
     /// Account id of the engine. It is expected to be `aurora`.
     aurora: AccountId,
+    /// Account id that will be used as admin for all deployed tokens.
+    token_admin: AccountId,
```

Comment:
Is there any way to access super-admins? I couldn't find any way right now. Anyway, having an explicit super-admin might still make sense, since this will be for the individual contracts and not for the factory.

Reply:
Good point, `acl_get_super_admins` is missing. I'll prepare a PR to add it to `near-plugins`.

Reply:
As agreed in our discussion, renamed the field token_super_admin to reflect its intention and be aligned with the naming used in near-plugins. Fixed in 6dc7d4850a0322c14719eb7c2265efc80449e9eb